packages/module/src/AssistantMessageEntry/AssistantMessageEntry.tsx

```diff
@@ -52,15 +55,22 @@ interface AssistantMessageEntryProps {
     props?: LabelProps;
   }[];
   icon?: React.ComponentType;
+  dropdown?: {
+    label: string;
```

Comment:
It might be better to have `ReactNode` instead of `string` so apps using this extension can pass custom elements to it as well.

Reply:
Looking at the implementation it might be better to pass object into the `dropdown` which would hold the dropdown props. For instance if extension user would like to adjust the dropdown.
```JSX
dropdown?: {
  dropdownProps?: DropdownProps;
  items: { label: React.ReactNode; props?: DropdownItemProps }
}
```

@josejulio what do you think about this?

Reply:
It makes sense to me- I'm happy that we already hold data to customize each item. 
Allowing to pass any react node in the label and further specify the dropdown props would be great!

Reply:
@josejulio @karelhala I've made the suggested changesto the prop. Let me know if there's more